app/client/packages/dsl/src/migrate/migrations/089-migrate-jsonformwidget-labelposition-and-alignment.ts

```diff
@@ -0,0 +1,48 @@
+import type { DSLWidget, WidgetProps } from "../types";
+import { traverseDSLAndMigrate } from "../utils";
+
+export const migrateJsonFormWidgetLabelPositonAndAlignment = (
+  currentDSL: DSLWidget,
+) => {
+  return traverseDSLAndMigrate(currentDSL, (widget: WidgetProps) => {
+    if (widget.type == "JSON_FORM_WIDGET") {
+      // console.log(`widget{{{{{{{{{{}}}}}}}}}}`,widget)
+      const jsonFormWidgetProps = widget;
+      Object.keys(jsonFormWidgetProps.schema).forEach((key) => {
+        const field = jsonFormWidgetProps.schema[key];
```

Comment:
```suggestion
Object.entries(jsonFormWidgetProps.schema).forEach((key, field) => {
```

This way you don't need an extra declaration.

Reply:
@rahulbarwal  this way I was unable to access the field children its taking it as a number field